Update Pupil Player docs to v2.0 (#379)
pupil-labs/pupil-docs: src/core/software/pupil-player.md

  <img src="../../media/core/imgs/pp-callout.jpg" style="display:flex;margin:0 auto;">
</div>

> 1. **Graphs**: This area contains performance graphs. The graphs `CPU`, `FPS`, and pupil algorithm detection confidence will be displayed.

Reviewer: ```suggestion
1. **Graphs**: This area contains performance graphs. The graphs display `CPU`, `FPS`, and pupil algorithm detection confidence.
```

### Analysis Plugins
These plugins are simple unique plugins, that operate on the gaze data for analysis and visualizations.

#### Surface Tracker

> This plugin is an post-hoc version of the [Surface Tracking](/core/software/pupil-capture/#surface-tracking) plugin for Pupil Capture.

Reviewer: ```suggestion
This plugin is a post-hoc version of the [Surface Tracking](/core/software/pupil-capture/#surface-tracking) plugin for Pupil Capture.
```

You can use this plugin to detect markers in the recording, define surfaces, edit surfaces, and create and export visualizations of gaze data within the defined surfaces.

<div class="pb-4">
  <img src="../../media/core/imgs/offline-srf-tracker.jpg" style="display:flex;margin:0 auto;">
</div>

Here is an example workflow for using the `Surface Tracker` plugin to generate heatmap visualizations and export surface data reports:

>   + Load `Surface Tracker` plugin - if you already have surfaces defined, the load may take a few seconds because the plugin will look through the entire video and cache the detected surfaces.

Reviewer: ```suggestion
  + Load the `Surface Tracker` plugin - if you already have surfaces defined, the load may take a few seconds because the plugin will look through the entire video and cache the detected surfaces.
```
